src/main/java/seedu/duke/command/ListAndStatsCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.duke.command;
+
+//@@author chydarren
+import seedu.duke.data.TransactionList;
+import seedu.duke.data.transaction.Transaction;
+import seedu.duke.exception.GlobalMissingPeriodNumberTagException;
+import seedu.duke.exception.GlobalMissingYearTagException;
+import seedu.duke.exception.GlobalUnsupportedTagException;
+import seedu.duke.exception.MoolahException;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public abstract class ListAndStatsCommand extends Command {

```

Comment:
Great use of inheritance to extract the common functions between list and stats!